MOBT-2-Swift-Foundation.playground/Contents.swift

```diff
@@ -0,0 +1,332 @@
+import Foundation
```

Comment:
Even in playground sometimes it's a good idea to break down code into multiple files (not critical here)